### ISSUE TITLE
Possible to enhance FileTree to hold multi folder as TVI_ROOT?

### ISSUE THREAD

**Issue by kitepad** [stefankueng/BowPad#214]
Many modern ide/editor can open a folder as workspace. My thought is :
1. BowPad can open many folder by "Add Folder".
2. Each added folder insert into FileTree as a root item.
3. Click file can open in tab.
4. Tab switching can reflect in file tree.
5. FileTree Prototype image as below: 
   
![image](https://user-images.githubusercontent.com/28701482/136658583-a9b3e804-546e-4d86-b6f0-b900661e7820.png)

**Comment by stefankueng** [comment]
the file tree shows the folder of the currently active tab - they don't have to be from the same subfolder. with your suggestion this would get very, very messy!
Just imagine having 20 tabs opened, all from different folders.

**Comment by wmjordan** [comment]
> Just imagine having 20 tabs opened, all from different folders.

Yes, I can have that when I am working with many configuration files and log files on a server.
There will be too many items for me.

**Comment by kitepad** [comment]
Understand.

**Comment by kitepad** [comment]
Can we show current active tab's folder as Tree root? Otherwise, I have to move my eyes to title bar to find current folder name. In my daily works, some opened files have same name, for example, three rust projects have three cargo.toml. I need to modify them base on their location.
